### PR TITLE
CS-5650 - Topics API - implement additional endpoints/methods & CS-5664

### PR DESCRIPTION
**Pull request by takeit**
- [x] - CS-5650 - Topics API - implement additional endpoints/methods
- [x] - CS-5664 - Chinese and Russia are not supported by Editorial Comments
- [x] - behat test
- [x] - phpspec specs
- [x] - install postfix on travis

### CODE REVIEW

**Comment by ahilles107** [comment]
how about unlinking?

**Comment by takeit** [comment]
its there now :)

File: newscoop/src/Newscoop/GimmeBundle/Controller/UserTopicsController.php [hunk 1]
```diff
@@ -0,0 +1,166 @@
...
+     *     },
+     *     parameters={
+     *         {"name"="language", "dataType"="string", "required"=false, "description"="Language code"}
+     *     },
+     *     requirements={
+     *         {"name"="id", "dataType"="integer", "required"=true, "description"="User Id"}
+     *     }
+     * )
+     *
+     * @Route("/userTopics/{id}.{_format}", defaults={"_format"="json"}, options={"expose"=true})
```
Comment (ahilles107) [inline newscoop/src/Newscoop/GimmeBundle/Controller/UserTopicsController.php]: i don't like it. 

why not ```/user/{id}/topics```

File: newscoop/src/Newscoop/GimmeBundle/Controller/UserTopicsController.php [hunk 2]
```diff
@@ -0,0 +1,166 @@
...
+     *     requirements={
+     *         {"name"="id", "dataType"="integer", "required"=true, "description"="User Id"}
+     *     }
+     * )
+     *
+     * @Route("/users/{id}.{_format}", defaults={"_format"="json"}, options={"expose"=true})
+     * @Method("LINK")
+     * @View(statusCode=201)
+     */
+    public function linkTopicToUserAction(Request $request, $id)
```
Comment (ahilles107) [inline newscoop/src/Newscoop/GimmeBundle/Controller/UserTopicsController.php]: Action name should be more generic like: ```linkToUserAction```.

**Comment by ahilles107** [comment]
:+1:

**Pull request merged**